Gopkg.lock

```diff
@@ -145,7 +145,10 @@
 
 [[projects]]
   name = "github.com/magiconair/properties"
```

Comment:
Note you got the wrong assert package! we need github.com/stretchr/testify/assert